src/explorerCommands/delete.ts

```diff
@@ -23,16 +23,24 @@ export async function deleteFiles(node?: DataNode): Promise<void> {
     );
 
     if (answer === confirmMessage) {
-        workspace.fs.delete(Uri.parse(node.uri), {
-            recursive: true,
-            useTrash: true,
-        });
+        try {
+            await workspace.fs.delete(Uri.parse(node.uri), {
+                recursive: true,
+                useTrash,
+            });
+        } catch (e) {
```

Comment:
Does the error mentioned in https://github.com/microsoft/vscode-java-dependency/issues/608#issuecomment-1097526210 will still happen even if we restrict it via `explorerResourceMoveableToTrash`?

Reply:
ok, let's leave it here and monitor the telemetry.